packit_service/service/api/projects.py

```diff
@@ -78,6 +78,41 @@ def get(self, forge, namespace, repo_name):
         return response_maker(project_info)
 
 
+@ns.route("/<forge>")
+@ns.param("forge", "Git Forge")
+class ProjectsForge(Resource):
+    @ns.expect(pagination_arguments)
+    @ns.response(HTTPStatus.PARTIAL_CONTENT.value, "Projects list follows")
+    @ns.response(HTTPStatus.OK.value, "OK")
+    def get(self, forge):
+        """List of projects of given forge"""
```

Comment:
Could you please provide a few examples for the forge argument? So we can have them displayed in the docs: https://prod.packit.dev/api/